vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/boutiquesParsing/BoutiquesDescriptor.java

```diff
@@ -0,0 +1,166 @@
+package fr.insalyon.creatis.vip.application.client.view.boutiquesParsing;
+
+import com.google.gwt.json.client.JSONArray;
+import com.google.gwt.json.client.JSONObject;
+import com.google.gwt.json.client.JSONParser;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Representation of an application Boutiques descriptor
+ *
+ * @author Guillaume Vanel
+ * @version %I%, %G%
```

Comment:
It's supposed to be substituted by version and date. I found it used on several different pages, but the only explanation I found was on this one at paragraph @version and in the bootm of the web page:
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#@version

Have no idea if it is good or bad practice

Reply:
https://stackoverflow.com/questions/578363/javadoc-version-and-since

That dates from 2009 and I can not find more recent recommendations, especially ones for git. I think it is outdated and should not be used anymore.

Reply:
Should I remove them all or simply put a real version number and date ?